src/services/NexusUploader.ts

```diff
@@ -20,7 +20,15 @@ export class NexusUploader {
     classifier: Classifier
   ): Promise<void> {
     const pkgJson = this.getPackageJson(appPath);
-    await this.uploadToNexus(pkgJson, version, classifier);
+    const hasDependencies = Object.keys(pkgJson.dependencies).length > 0;
```

Comment:
Du må sjekke om bundledDependencies har elementer siden det er den listen som blir pakket samme. Ser i metoden (this.getPackageJson) må kalle opp createPackageWithBundledDeps isteden for getPackageJson fra packageJson.ts.
Den slår sammen alle relevante dependencies som skal pakkes sammen.